.github/workflows/release.yml

```
@@ -27,13 +27,13 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: 14
+          node-version: 20
 
       - name: Install dependencies
         run: npm ci
 
       - name: Release
-        run: npx semantic-release@v19.0.5
+        run: npx -y semantic-release@24.1.2
```

Let's specify a range so that we can receive patches.
```suggestion
        run: npx -y semantic-release@^24
```